src/main/archetype/pom.xml

```diff
@@ -187,7 +187,7 @@
                             </goals>
                         </execution>
                         <execution>
-                            <id>npm install</id>
+                            <id>npm ci</id>
```

Comment:
@vladbailescu this doesn't actually work, the default command for goal npm is still install:

https://github.com/adobe/aem-project-archetype/actions/runs/3089250468/jobs/4996652096#step:6:619

We should do

```
<execution>
    <id>npm ci</id>
    <goals>
        <goal>npm</goal>
    </goals>
    <configuration>
        <arguments>ci</arguments>
    </configuration>
</execution>
```